openml-lightgbm/lightgbm-builder/javadoc/README.md

```diff
@@ -0,0 +1,2 @@
+# Ligthgbm Lib Javadoc
+This is a dummy javadoc.
```

Comment:
Do we need this? If so move on :)

Reply:
Yes. The previous [PR](https://github.com/feedzai/feedzai-openml-java/pull/46) was made due to javadoc and source error. See its [description](https://github.com/feedzai/feedzai-openml-java/pull/46)